backend/src/utils.ts

```diff
@@ -11,7 +11,7 @@ const typeToElStrMap = {
   'heading-two': 'h2',
   'bulleted-list': 'ul',
   'numbered-list': 'ol',
-  'paragraph': 'p',
+  paragraph: 'p',
```

Comment:
Prettier does this automatically since regular keys don't need quotes, but keys with dashes need quotes